android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -12,6 +12,8 @@
 import android.os.Bundle;
 
 public class TagDispatcher {
+    private static final int DELAY_PRESENCE = 5000;
```

Comment:
@adriaan-telcred Please see: https://code.google.com/p/android/issues/detail?id=58773

We would need to identify if the phone is using a broadcom chipset to be able to deactivate this timeout.